build.gradle

```diff
@@ -178,6 +178,9 @@ dependencies {
 
     compile 'com.google.code.findbugs:jsr305:1.3.9' // Annotations used by google libs.
 
+    //EDITS for Flaxbeard's Steam Power
+    compile files('Flaxbeard.jar')
```

Comment:
Please add the dependency in a way it can be autodownloaded. If FSP has a maven, that'd be ideal, otherwise add it the the CoFH stuff is referenced, e.g. (i.e. via Curseforge).